crates/fluvio-cli/src/produce/mod.rs

```diff
@@ -161,6 +167,16 @@ impl ProduceOpt {
             _ => config_builder,
         };
 
+        // Delivery Semantic
+        let config_builder = if let Some(delivery_semantic) = self.delivery_semantic {
+            if delivery_semantic == DeliverySemantic::AtMostOnce && self.isolation.is_some() {
+                warn!("Isolation is ignored for AtMostOnce delivery semantic");
```

Comment:
@morenol I think there is way to enforce at clap using conditional flag?

Reply:
I think that we can enforce at clap level with `requires` when we want to enable/disable a parameter depending on the presence of other, but since this one is a enum type, I think that we cannot use that.

https://github.com/clap-rs/clap/blob/ef3121cbdf938fef9a6b2dc199bc15998043939a/examples/tutorial_derive/04_03_relations.rs#L35

Maybe an option is to use `--at-most-once-delivery` and `--at-least-once-delivery` instead of the enum type, but not sure if that is better for the CLI users